aopy/preproc/bmi3d.py

```diff
@@ -723,4 +725,55 @@ def get_target_events(exp_data, exp_metadata):
         event_target = target_location[None,:] * target_on[:,None]    
         target_events.append(event_target)
         
-    return np.array(target_events).transpose(1,0,2)
\ No newline at end of file
+    return np.array(target_events).transpose(1,0,2)
+
+def get_ref_dis_frequencies(data, metadata):
+    '''
+    For continuous tracking tasks, get the set of frequencies (in Hz) used to 
+    generate the reference and disturbance trajectories that were preesented 
+    on each trial of the experiment.
+
+    Args:
+        data (dict): A dictionary containing the experiment data.
+        metadata (dict): A dictionary containing the experiment metadata.
+
+    Returns:
+        tuple: Tuple containing:
+            | **freq_r (list of arrays):** (ntrial) list of (nfreq,) frequencies used to generate reference trajectory
+            | **freq_d (list of arrays):** (ntrial) list of (nfreq,) frequencies used to generate disturbance trajectory
+    '''
```

Comment:
can you add the examples and images from your test function here? that way they will show up in readthedocs. see https://analyze.readthedocs.io/en/latest/_modules/aopy/analysis/base.html#calc_cwt_tfr for an example